.github/workflows/conformance-eks.yaml

```diff
@@ -202,6 +198,13 @@ jobs:
         run: |
           kubectl -n kube-system delete daemonset aws-node
 
+      - name: Create IPsec key
+        if: ${{ matrix.ipsec == true }}
+        shell: bash
+        run: |
+          KEYID=$((1 + $RANDOM % 15))
```

Comment:
Please no :pleading_face: No randomness in CI. If you want to test both 1->2 and 15->1 then let's do that explicitly.